api/utilities.py

```diff
@@ -1,10 +1,34 @@
-
+from requests import get, post
 import facebook as fb
 from django.http import HttpResponse
-
 import os
 from dotenv import load_dotenv
 
+def create_discord_event(name, description, start_time, end_time, location, image):
+    acm_guild_id = ""
+    acm_bot_token = ""
+    api_url = "https://discord.com/api/guilds/" + acm_guild_id + "/scheduled-events"
+
+    data = {
+        'name': name,
+        'description': description,
+        'privacy_level': '2',
+        'scheduled_start_time': start_time,
+        'scheduled_end_time': end_time,
+        'entity_type': '3',
+        'entity_metadata': {'location': location},
+        #'image': image,
+    }
+
+    headers = {
+        'Authorization': 'Bot %s' % (acm_bot_token),
+        'Content-Type': 'application/json',
+    }
+
+    created_event = post(api_url, json = data, headers = headers).json()
```

Comment:
Same comment as with the instagram function PR, I think we should wrap the post request into a try-catch block to handle any external discord API errors or rate limit errors on our end!